archway-1/chain.json

```diff
@@ -241,6 +254,10 @@
       {
         "address": "archway-grpc.openbitlab.com:443",
         "provider": "openbitlab"
+      },
+      {
+        "address": "https://grpc-archway.mainnet.validatrium.club",
```

Comment:
This URL does  answer

Reply:
Upgraded grpc link

Reply:
Still getting:
```
grpcurl grpc-archway.mainnet.validatrium.club:443 list
Failed to dial target host "grpc-archway.mainnet.validatrium.club:443": dial tcp 135.181.58.28:443: connect: connection refused
```

Reply:
and the conflicts need to be fixed as well :)

Reply:
Conflicts are resolved

Reply:
> Still getting:
> 
> ```
> grpcurl grpc-archway.mainnet.validatrium.club:443 list
> Failed to dial target host "grpc-archway.mainnet.validatrium.club:443": dial tcp 135.181.58.28:443: connect: connection refused
> ```
```bash
grpcurl -plaintext grpc-archway.mainnet.validatrium.club:27490 list
archway.rewards.v1.Query
archway.tracking.v1.Query
...
```